### ISSUE TITLE
error while mixing, after downloading sources

### ISSUE THREAD

**Issue by HiddenTrackMJ** [JorisCos/LibriMix#17]
hello !  this happened when i ran gen_xxx.sh  (i've got dev_clean, test_clean, clean100, clean360).  please help me !
![image](https://user-images.githubusercontent.com/41421003/122359034-c8b87b00-cf87-11eb-8ef0-e1db40349230.png)

**Comment by HiddenTrackMJ** [comment]
lol, i find the problem .
![image](https://user-images.githubusercontent.com/41421003/122369646-f655f200-cf90-11eb-87bb-3e509db1320f.png)

**Comment by Ashh-Z** [comment]
@HiddenTrackMJ how did you solve this problem ? I am facing a similar issue

**Comment by hashirkz1** [comment]
> @HiddenTrackMJ how did you solve this problem ? I am facing a similar issue

did you find the solution i am facing the same issue
please help

**Comment by Ashh-Z** [comment]
> > @HiddenTrackMJ how did you solve this problem ? I am facing a similar issue
> 
> did you find the solution i am facing the same issue please help

In my case, some of the folders were not downloading, as a result of which I was getting a similar error. This was an issue on their side and once it was fixed all the folders were downloading fine and I did not face any further issues. I had previously created an issue for this https://github.com/JorisCos/LibriMix/issues/26#issue-2126700695. 

I suggest you also check if all the files are getting downloaded or not.
